src/featureFlags.js

```diff
@@ -21,7 +22,9 @@ export class FeatureFlags extends events.EventEmitter {
     this.userId = userId;
     this.network = network;
     this.walletServiceFlag = WALLET_SERVICE_FEATURE_TOGGLE;
+    this.atomicSwapFlag = `atomic-swap-wallet-desktop-${this.network}.rollout`;
```

Comment:
This is the old feature flag model, the new one does not need `network`. You can do similar to the wallet service feature flag and create the variable in the constants file.